modules/ossm-vs-istio.adoc

```diff
@@ -136,4 +88,4 @@ All OpenShift routes created by {ProductName} are in the `istio-system` namespac
 
 [id="ossm-cni_{context}"]
 == Container Network Interface (CNI)
```

Comment:
Add "Plugin" (and possibly "Istio" too - e.g. "Istio Container Network Interface (CNI) Plugin")

Reply:
Fixed.